test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -802,6 +802,500 @@ test.describe( 'Navigation block', () => {
 	} );
 } );
 
+test.describe( 'Navigation block - Frontend interactivity', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'emptytheme' );
+		await requestUtils.deleteAllTemplates( 'wp_template_part' );
+	} );
+
+	test.beforeEach( async ( { requestUtils } ) => {
+		await Promise.all( [ requestUtils.deleteAllMenus() ] );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await Promise.all( [
+			requestUtils.deleteAllMenus(),
```

Comment:
This one can be removed as it gets called in `afterEach`.

It would be easier to reason if database cleanup happen in a more organizes fashion like:
- delete all in `beforeAll`
- delete all in `afterEach`

or the other way around:
- delete all in `beforeEach`
- delete all in `afterAll`